client/layout/masterbar/checkout.jsx

```diff
@@ -72,4 +84,9 @@ class CheckoutMasterbar extends React.Component {
 	}
 }
 
-export default connect( null, { recordTracksEvent } )( localize( CheckoutMasterbar ) );
+export default connect(
+	( state ) => ( {
+		visitingFromJetpackCloud: getInitialQueryArguments( state ).source === 'jetpack-plans',
```

Comment:
We need to include `'jetpack-connect-plans'` too since it's used in the redirection at the end of the in-place connection flow.

Reply:
Good catch, thank you.